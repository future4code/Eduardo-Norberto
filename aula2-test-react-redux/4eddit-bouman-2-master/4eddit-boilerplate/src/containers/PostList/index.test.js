import React from "react"
import renderer from "react-test-renderer"
import { PostDiv, PostTittle } from "./index"
import { shallow } from "enzyme"


describe("Testa o componente PostCreate", ()=> {
    test('Snapshot', ()=>{
        
        const listPost = renderer.create(<PostDiv />).toJSON();
        expect(listPost).toMatchSnapshot();
    })

    // test('Testa se os componentes PostCreate sÃ£o renderizados', ()=>{
    //     const mockCreatePost =  {title: "Teste1"}
        

    //     const component = shallow(<PostTittle posts={mockCreatePost} />)

    //     const allItems = component.find(PostTittle)

    //     expect(allItems.props().title).toBe( "Teste1")
    // })
})