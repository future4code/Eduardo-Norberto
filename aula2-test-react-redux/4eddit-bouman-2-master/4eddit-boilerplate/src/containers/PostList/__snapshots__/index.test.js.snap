// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testa o componente PostCreate Snapshot 1`] = `
<div
  className="sc-fzXfMv dajeqe"
/>
`;
